spec/datetimeformat.html

```diff
@@ -81,6 +81,12 @@
         1. Let _opt_ be a new Record.
         1. Let _matcher_ be ? GetOption(_options_, *"localeMatcher"*, *"string"*, &laquo; *"lookup"*, *"best fit"* &raquo;, *"best fit"*).
         1. Set _opt_.[[localeMatcher]] to _matcher_.
+        1. Let _calendar_ be ? Get(_options_, *"calendar"*).
+        1. If _calendar_ is not *undefined*, set _calendar_ to ? ToString(_calendar_).
+        1. Set _opt_.[[ca]] to _calendar_.
+        1. Let _numberingSystem_ be ? Get(_options_, *"numberingSystem"*).
+        1. If _numberingSystem_ is not *undefined*, set _numberingSystem_ to ? ToString(_numeringSystem_).
```

Comment:
Typo: numeringSystem

Reply:
@littledan can you update the PR to fix these typos?

Reply:
Fixed the typo, apologies for the delay.